calico/felix/actor.py

```diff
@@ -232,11 +249,28 @@ def _step(self):
                 except BaseException as e:
                     _log.exception("Exception processing %s", msg)
                     results.append(ResultOrExc(None, e))
+                    message_exception = e.__class__.__name__ +\
+                                                      ': ' + str(e)
                     _stats.increment("Messages executed with exception")
                 else:
+                    message_exception = None
                     results.append(ResultOrExc(result, None))
                     _stats.increment("Messages executed OK")
                 finally:
+                    batch_uuids.add(msg.uuid)
+                    # Log the message info
```

Comment:
These comments aren't so useful.  Why are you logging the message info and what's special about this log?  Also, there's no need to describe JSON as "Strings are enclosed in quotes. Numbers aren't.", you can call it JSON.